lib/utils/utils.ts

```diff
@@ -18,3 +19,37 @@ export default function makeRoute() {
     },
   }).use(cors());
 }
+
+export function getImageBufferFromBase64(base64string: string): Buffer {
+  const cleanedBase64String: string = base64string.replace(/^data:image\/\w+;base64,/, '');
```

Comment:
‼️ Regex is scary, how do you know this works? What assumptions are you making about the input string? Document this if you can.

Reply:
will do. we can remove this from the PR since this is not a UI change, which are the only critical change (upcoming deadline). thoughts?